vivisect/tools/graphutil.py

```diff
@@ -622,35 +662,38 @@ def preRouteGraph(graph, fromva, tova):
     preRouteGraphUp(graph, tova)
     preRouteGraphDown(graph, fromva)
 
-def clearGraphRouting(graph):
```

Comment:
allow for clearing of specific tags, change graph routing to apply specific tags for up and down, and handle looping